packages/commonwealth/client/vite.config.ts

```diff
@@ -122,6 +122,10 @@ export default defineConfig(({ mode }) => {
         env.FLAG_KNOCK_PUSH_NOTIFICATIONS_ENABLED,
       ),
       'process.env.FLAG_CONTEST_DEV': JSON.stringify(env.FLAG_CONTEST_DEV),
+      'process.env.ETH_ALCHEMY_API_KEY':
+        JSON.stringify(env.ETH_RPC).trim() === '"e2e-test"'
```

Comment:
```suggestion
        env.ETH_RPC === 'e2e-test' && env.NODE_ENV === 'test'
```

Reply:
I couldn't get the strings to match without trimming, but the other suggestion is implemented